be/src/olap/rowset/segment_v2/binary_plain_page.h

```diff
@@ -123,13 +136,26 @@ class BinaryPlainPageBuilder : public PageBuilder {
     }
 
 private:
+    Status _get_value_at(void* value, size_t idx) const {
+        if (_offsets.size() == 0) {
+            return Status::NotFound("page is empty");
+        }
+        size_t value_size = (idx < _offsets.size() - 1) ?
+            _offsets[idx + 1] - _offsets[idx] : _last_value_size;
```

Comment:
Right now each item's start offset is stored in _offsets[item_idx]. To get the size of last item, there are two ways depending on whether we're building the page or we have built the page

- When the page is being built, `add()` should remember the size of last value in `_last_value_size`
- When the page has been built, it can be computed as `trailer_start_offset - offsets[num_entries - 1]`

Reply:
OK